scripts/app.js

```diff
@@ -269,15 +269,42 @@ function shackUp() {
 
 
 	this.sanitizeListings = function(data) {
-		return _.map(data.listings, function(listing) {
+		return _.compact(_.map(data.listings, function(listing) {
 			listing.city = '';
+			listing.agentName = 'Agent Contact';
+			listing.agentPhone = '';
+			listing.agentEmail = '';
+			listing.agentPhoto = 'http://sunfieldfarm.org/wp-content/uploads/2014/02/profile-placeholder.png';
 
+
+			// listing city
 			if (listing.address.AddrCity) {
 				listing.city = listing.address.AddrCity;
 			} else if (listing.address.AddrCounty) {
 				listing.city = listing.address.AddrCounty;
 			}
-		});
+
+			// listing agent phone + photo
+			if (listing.agents[0]) {
+				if (listing.agents[0].phones) {
+					listing.agentPhone = listing.agents[0].phones[0];
+				}
+				if (listing.agents[0].photo) {
+					listing.agentPhoto = listing.agents[0].photo;
+				}
+				if ( listing.agents[0].name ) {
+					listing.agentName = listing.agents[0].name;
+				}
+				if ( listing.agents[0].email ) {
+					listing.agentEmail = listing.agents[0].email;
+				}
+			}
+
+			// exclude listings w/o images
+			if (listing.photos) {
```

Comment:
It'd be nice to return without doing any of the other stuff if there aren't photos